fe/fe-core/src/main/java/com/starrocks/qe/StmtExecutor.java

```diff
@@ -1134,6 +1134,9 @@ public void handleDMLStmt(ExecPlan execPlan, DmlStmt stmt) throws Exception {
         TransactionStatus txnStatus = TransactionStatus.ABORTED;
         try {
             if (execPlan.getFragments().get(0).getSink() instanceof OlapTableSink) {
+                //if sink is OlapTableSink Assigned to Be execute this sql [cn execute OlapTableSink will crash]
+                context.getSessionVariable().setPreferComputeNode(false);
+                context.getSessionVariable().setUseComputeNodes(0);
```

Comment:
setUseComputeNodes(0) is enough?

Reply:
The semantic here is not to use Compute node. Semantically, it is appropriate to set setPreferComputeNode(false), although the effect is the same without setting it now, but it can be unaffected when these two parameters are modified later, such as adding After the label function, you may want to throw an exception instead of using Backend, so that this block will have an impact if setPreferComputeNode(false) is not set.